modules/recon/hosts-hosts/hackertarget_reverse.py

```diff
@@ -0,0 +1,36 @@
+from recon.core.module import BaseModule
+
+
+class Module(BaseModule):
+    meta = {
+        'name': 'HackerTarget reverse IP lookup',
+        'author': 'Qazeer',
+        'version': '1.0',
+        'description': 'Uses the HackerTarget.com reverse IP lookup API to find host names associated with IP addresses. Updates the \'hosts\' table with the results.',
+        'query': 'SELECT DISTINCT ip_address FROM hosts WHERE ip_address IS NOT NULL',
+    }
+
+    def module_run(self, hosts):
+        for ip in hosts:
+            self.heading(ip, level=0)
+            url = 'https://api.hackertarget.com/reverseiplookup/'
```

Comment:
Since this is static, it can be set outside of the loop so it doesn't get executed for every IP.